app/models/user.rb

```diff
@@ -222,16 +222,10 @@ def formatted_reputation(*args)
   end
 
   memoize
-  def active_subscription? = subscriptions.active.exists?
+  def current_subscription? = current_subscription.present?
```

Comment:
I think we still want `active_subscription` for use in `app/commands/user/insiders_status/determine_eligibility_status.rb`

Reply:
(In fact, I don't understand how this passes the tests with it removed!)

Reply:
`active_subscription` in `determine_eligibility_status` does not use the method on user :)

Reply:
Ah!